demos/smartlab_demo/python/object_detection/postprocess.py

```diff
@@ -0,0 +1,45 @@
+import torch
+import torchvision
```

Comment:
Remove `torch` and `torchvision` dependencies

Reply:
removed torch dependency